README.md

```diff
@@ -396,6 +396,259 @@ public CloudTasksClient getService() throws IOException {
 }
 ```
 
+## Retries
+Client Libraries use retries to handle unexpected, transient failures (i.e. Server temporarily unavailable).
+Multiple attempts, hopefully, will result in a successful response from the server.
+
+By default, retries are configured by the Cloud Service. The configured retry parameters are defined per RPC.
```

Comment:
Would you add an example (the audience is client library users)?

Reply:
From the audience's perspective, it's just retries are controlled by the responses of the service.

> The configured retry parameters are defined per RPC.

If the audience doesn't have a way to read the definition, let's remove this.

Reply:
> Would you add an example (the audience is client library users)?

I'll move the Java-Asset example up to show the one service's default retry param values

> From the audience's perspective, it's just "retries are controlled by the responses of the service." isn't it?

Controlled by response of the service and the time/ attempt bounds.